st-tech/fluent-bit-pubsub-custom
# Modify Test Method

## Summary
- Enabled tests to read environment variables from `.env` file.
- Removed the previous test method using CircleCI.

## Changes
- Integrated `godotenv` to load environment variables from `.env` file in test cases.
- Updated test code to read project ID and topic name from environment variables.
- Deleted CircleCI configuration files.

## Verification Steps
1. Create a `.env` file in the project's root directory and set the necessary environment variables.
2. Run tests and ensure all tests pass.

## Additional Information
- None
